src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double katetA;
+    private final double katetB;
+
+    public RightTriangle(double katetA, double katetB) {
+        this.katetA = katetA;
+        this.katetB = katetB;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: right triangle, area: " + obtainTheArea()
+                + " sq. units, katet A: " + katetA + " units, katet B: " + katetB
+                + " units, color: " + getColor());
```

Comment:
The method name `toDraw` suggests an action, but it's in a class, which should be a noun. Consider renaming it to better reflect its purpose, like `drawInfo` to align with the `DrawInfo` interface.